code/modules/atmospherics/machinery/pipes/layeredmanifold4w.dm

```diff
@@ -0,0 +1,315 @@
+/* Four-way manifolds */
+
+/*===== Haute tayau *kisses fingers* ===== */
```

Comment:
What.

Please add proper comments explaining the function of the files all over the place.

Reply:
_Quality pipe_